roles/infra/README.md

```diff
@@ -197,6 +197,7 @@ More information about the parameters can be found in the [ovirt_host_network](h
 
 ### Storages
 To setup storages you can define dictionary variable called `storages`.
+This role supports only storages with one connection.
```

Comment:
maybe you can add something like:
"In case of more than one connection, the storage connection update of this domain will be skipped."

Reply:
+1

Reply:
+1